apps/apps.c

```diff
@@ -2294,29 +2294,28 @@ int app_access(const char* name, int flag)
 #ifdef _WIN32
 int app_isdir(const char *name)
 {
-    HANDLE hList;
-    WIN32_FIND_DATA FileData;
+    DWORD attr;
 # if defined(UNICODE) || defined(_UNICODE)
     size_t i, len_0 = strlen(name) + 1;
+    WCHAR tempname[MAX_PATH];
 
-    if (len_0 > OSSL_NELEM(FileData.cFileName))
+    if (len_0 > MAX_PATH)
```

Comment:
Formally speaking there is no reason to hold on to the MAX_PATH limitation. I mean in FindFirstFile you have no choice, but here you do. But in this case, I'd say it's ok...